src/renderer/styles/img/lock-black.svg

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```

Comment:
You might want to use another icon of yourselves as the default, maybe even a Buttercup icon?
Just in case this would stay, it comes from http://flaticon.com, so would need to give credit on the `README`

Reply:
Thanks @hugo-cardenas - Good idea. I think we'll definitely use our own. What icon do you think we should use @sallar?

Reply:
@hugo-cardenas can I get a screenshot of this in action?

Reply:
Maybe you have already seen it but:
<img width="910" alt="screen shot 2017-12-11 at 22 23 26" src="https://user-images.githubusercontent.com/1535759/33942654-404f59d8-e01f-11e7-978a-e5494ddd6493.png">

Reply:
Wow! I love it.

I would perhaps reduce the gap between the icons and the text a _tiny_ bit.. and maybe center the icon a bit more.. but it's looking very elegant.

Reply:
@hugo-cardenas you can use this lock from Google Material Icon Set which we also use in Buttercuo Mobile:
[ic-lock-48px.svg.zip](https://github.com/buttercup/buttercup-desktop/files/1556237/ic-lock-48px.svg.zip)